components/views/services/Services.styles.tsx

```diff
@@ -1,159 +1,118 @@
+        flex-direction: column;
+        row-gap: 4rem;
+        @media only screen and (max-width: 1350px) {
+            padding: 1rem;
         }
-    }
 
-    @media only screen and (max-width: 1350px) {
         .detail-section {
             display: flex;
-            flex-direction: column;
-            row-gap: 1.6rem;
+            column-gap: 2rem;
             align-items: center;
-
-            width: var(--max-desktop-width);
-            padding: var(--content-padding);
+            width: initial;
+            @media only screen and (max-width: 1350px) {
+                flex-direction: column;
+                row-gap: 1.6rem;
+                align-items: center;
+                padding: 0;
+            }
             img {
-                height: 15rem;
-                width: 15rem;
+                height: 9rem;
+                width: 9rem;
             }
 
             .service-info {
-                border-top: 3px solid var(--Secondary-accent-color);
-                border-left: 3px solid transparent !important;
-                padding-left: 0 !important;
-                display: flex;
-                flex-direction: column;
-                row-gap: 1.6rem;
+                border-left: 3px solid var(--Secondary-accent-color);
+                padding-left: 2rem;
 
                 h1 {
-                    text-align: center;
-                    font-size: 4rem;
-                }
-
-                .description {
-                    text-align: center;
+                    font-size: 2.5rem;
                 }
 
                 ul {
-                    font-size: 1.6rem;
+                    font-size: 1rem;
                     list-style-type: none;
                     display: grid;
                     display: grid;
-                    justify-items: start;
                     grid-template-columns: repeat(
                         auto-fill,
-                        minmax(200px, 1fr)
-                    ) !important;
-                    grid-template-rows: repeat(auto-fill, minmax(20px, 1fr));
-                    row-gap: 0.64rem;
-                    padding-bottom: 2.56rem;
+                        minmax(350px, 1fr)
+                    );
+                    grid-template-rows: repeat(auto-fill, minmax(10px, 1fr));
+                    row-gap: 0.4rem;
+
+                    padding: 1rem 0rem;
                     li {
-                        margin-left: 2.4rem;
+                        margin-left: 1.5rem;
                         ::before {
                             content: '\\2713';
                             color: var(--Secondary-accent-color);
-                            margin-left: -1.92rem;
-                            padding-right: 0.8rem;
+                            margin-left: -1.2rem;
+                            padding-right: 0.5rem;
                         }
                     }
                 }
+                @media only screen and (max-width: 1350px) {
```

Comment:
Same here. Move these styles to within the media query below. The only time you want to include the responsive styles within a class like this is if it's the only time/reason the media query is being used within the file